src/pybind/mgr/openattic/module.py

```diff
@@ -36,7 +47,14 @@ def shutdown(self):
         cherrypy.engine.stop()
 
     def handle_command(self, cmd):
-        pass
+        if cmd['prefix'] == 'openattic webgui set-user':
+            self.set_localized_config('username', cmd['username'])
+            hashed_passwd = bcrypt.hashpw(cmd['password'], bcrypt.gensalt())
```

Comment:
Do we need to preserve the generated salt value somewhere in order to be able to verify the password later?

Reply:
The salt is included in the hash: $bcrypt_id$log_rounds$128-bit-salt184-bit-hash.
It can simply compared with the following code
```
if bcrypt.hashpw(password, hashed) == hashed:
        print("It Matches!")
    else:
        print("It Does not Match :(")
```